tdei-api-gateway.json

```diff
@@ -370,28 +400,26 @@
                         "AuthorizationToken": []
                     }
                 ]
-            },
+            }
+        },
+        "/api/v1/osw/validate": {
             "post": {
                 "tags": [
-                    "GTFS-Pathways"
+                    "OSW"
                 ],
-                "summary": "create pathways file",
-                "description": "This call allows a user to upload or create a new gtfs pathways file. The caller must provide metadata about the file. Required metadata includes information about how and when the data was collected and valid dates of the file. Returns the tdei_record_id of the uploaded file.",
-                "operationId": "uploadPathwaysFile",
+                "summary": "Validates the osw dataset.",
+                "description": "Allows a user to validate osw dataset to check the correctness of data. The caller must provide metadata about the file - includes information about how and when the data was collected and valid dates of the file. Returns the job_id associated with validation job.",
```

Comment:
we dont need the metadata for validating the dataset.

Reply:
Done